impl/c-qube/src/services/csv-adapter/parser/utils/csvreader.ts

```diff
@@ -11,7 +12,11 @@ export async function readCSV(filePath: string): Promise<string[][]> {
       .createReadStream(filePath)
       .pipe(csv({ separator: ',', headers: false, quote: "'" }))
       .on('data', (data) => {
-        rows.push(Object.values(data));
+        const rowValues: string[] = Object.values(data);
```

Comment:
what is this doing?

Reply:
how is this logic ensuring that case insesitivity is achieved? What if the dimensions table has a value "CQUBE" and `caseSensitiveFKSearch` flag is set to `false` and then the CSV has `cqube`, ideally as per the issue description this case should pass whereas in my understanding with the implementation in this PR, it will fail. Am I missing something here?